release-notes/swan-lake-alpha4-release-note.md

```diff
@@ -118,3 +175,15 @@ To view bug fixes, see the GitHub milestone for Swan Lake <VERSION> of the repos
 #### Ballerina Packages Updates
 
 ### Breaking Changes
+
+- Unused variables declared with `var` where the inferred type includes error result in a compilation error.
+- The `error<*>` syntax has been removed.
+- Relational expressions are no longer supported with numeric values when the static types of the operands belong to different ordered types.
+- The `indexOf` and `lastIndexOf` functions of the `lang.array` lang library cannot be used with values that do not belong to `anydata`.
+- An object used as the iterable value in a `foreach` statement or a `from` clause must be a subtype of `object:Iterable`.
```

Comment:
Shouldn't this include `join` clause as well?

Reply:
Yeah I think it should be included. @KRVPerera, can you please confirm? This seems to be the current behaviour in the implementation too, we should probably add a negative test if we don't have one already.

```suggestion
- An object used as the iterable value in a `foreach` statement, a `from` clause, or a `join` clause must be a subtype of `object:Iterable`.
```

Reply:
I agree with the change. BTW query expression is more generic.

Reply:
Changed in 67a18eb.

Reply:
Went with adding the join clause because we would have to mention both query expressions and actions separately if not.